src/main/groovy/org/shipkit/internal/gradle/versionupgrade/UpgradeDependencyPlugin.java

```diff
@@ -47,21 +48,21 @@
  *
  * Configure your 'shipkit.gradle' file like here:
  *
- *      apply plugin: 'org.shipkit.version-upgrade-consumer'
+ *      apply plugin: 'org.shipkit.upgrade-dependency'
  *
- *      versionUpgrade{
+ *      upgradeDependency{
  *          baseBranch = 'release/2.x'
  *          buildFile = file('gradle.properties')
```

Comment:
Let's use 'build.gradle' in the Javadoc example instead of 'gradle.properties'. The latter is very unconventional and I plan to get rid of the from Mockito.